pages/docs/mutation.en-US.md

```diff
@@ -5,7 +5,19 @@ SWR provides the `mutate` and `useSWRMutation` APIs for mutating remote data and
 ## mutate
 
 ```js
-const data = await mutate(key, data, options)
+import { mutate, useSWRConfig } from "swr"
```

Comment:
We should promote the `mutate` from `useSWRConfig` IMO to make sure `mutate` is really from the current context, bounded with the cache from that context.

```suggestion
import { mutate as globalMutate, useSWRConfig } from "swr"
```